plugin/src/main/groovy/com/novoda/staticanalysis/internal/detekt/DetektConfigurator.groovy

```diff
@@ -53,11 +54,8 @@ class DetektConfigurator implements Configurator {
         def detektTask = project.tasks['detektCheck']
         detektTask.group = 'verification'
 
-        // run detekt as part of check
-        project.tasks['check'].dependsOn(detektTask)
```

Comment:
This already transitively happens since `check` dependsOn `evaluateViolations`. No other tool configurator does this. Removed!